internal/release/services/release/release.go

```diff
@@ -54,6 +73,184 @@ func (s *ReleaseService) RegisterWithServer(grpcServer grpc.ServiceRegistrar) er
 	return nil
 }
 
+// Publishes a new release.
+func (s *ReleaseService) Publish(ctx context.Context, request *release.PublishRequest) (*release.PublishResponse, error) {
+
+	span := trace.SpanFromContext(ctx)
+	span.SetAttributes(
+		attribute.String("release.name", request.GetName()),
```

Comment:
Lets add "release.type" and "release.organization" here. I don't think we need links or description.